### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by yakov116**
Due to popular demand I have opened another lint PR. https://github.com/sindresorhus/refined-github/pull/3412#discussion_r4658459222

I changed the select.all's to (almost) always be an array. Its easier to read IMHO and gives the ability to add comments.

### CODE REVIEW

File: source/features/comment-fields-keyboard-shortcuts.tsx [hunk 1]
```diff
@@ -11,10 +11,10 @@ function eventHandler(event: delegate.Event<KeyboardEvent, HTMLTextAreaElement>)
...
 	if (event.key === 'Escape') {
 		// Cancel buttons have different classes for inline comments and editable comments
-		const cancelButton = select<HTMLButtonElement>(`
-				.js-hide-inline-comment-form,
-				.js-comment-cancel-button
-			`, field.form!);
+		const cancelButton = select<HTMLButtonElement>([
+			'.js-hide-inline-comment-form',
+			'.js-comment-cancel-button'
+		], field.form!);
```
Comment (fregante) [inline source/features/comment-fields-keyboard-shortcuts.tsx:17]: The only reason to do this is to add comments. Can you add comments?

File: source/features/keyboard-navigation.tsx [hunk 2]
```diff
@@ -6,10 +6,10 @@ import {isEditable} from '../helpers/dom-utils';
...
 const isCommentGroupMinimized = (comment: HTMLElement): boolean =>
 	select.exists('.minimized-comment:not(.d-none)', comment) ||
-	Boolean(comment.closest(`
-		.js-resolvable-thread-contents.d-none,
-		.js-resolvable-timeline-thread-container:not([open])
-	`));
+	Boolean(comment.closest([
+		'.js-resolvable-thread-contents.d-none',
+		'.js-resolvable-timeline-thread-container:not([open])'
+	].join()));
```
Comment (fregante) [inline source/features/keyboard-navigation.tsx:12]: Can you comment these?

File: source/features/comments-time-machine-links.tsx [hunk 3]
```diff
@@ -119,10 +119,10 @@ async function showTimemachineBar(): Promise<void | false> {
...
 function init(): void {
 	// PR reviews' main content has nested `.timeline-comment`, but the deepest one doesn't have `relative-time`. These are filtered out with `:not([id^="pullrequestreview"])`
-	const comments = select.all(`
-		:not(.js-new-comment-form):not([id^="pullrequestreview"]) > .timeline-comment:not(.rgh-time-machine-links),
-		.review-comment > .previewable-edit:not(.is-pending):not(.rgh-time-machine-links)
-	`);
+	const comments = select.all([
+		':not(.js-new-comment-form):not([id^="pullrequestreview"]) > .timeline-comment:not(.rgh-time-machine-links)',
+		'.review-comment > .previewable-edit:not(.is-pending):not(.rgh-time-machine-links)'
+	]);
```
Comment (fregante) [inline source/features/comments-time-machine-links.tsx:125]: Does this need to be commented? Can you add comments if they help?

File: source/features/index.tsx [hunk 4]
```diff
@@ -93,6 +93,10 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 		return;
 	}
 
+	if (!document.head.childElementCount) {
```
Comment (yakov116) [inline source/features/index.tsx:96]: On any of the pages that "were not supposed to be on" like https://github.com/sindresorhus/refined-github/pull/3433/show_toc?base_sha=784cb0c745fba344a24c3bcc9d96092a1a1ce166&sha1=784cb0c745fba344a24c3bcc9d96092a1a1ce166&sha2=ed17f32266803e38aaea74517bdb0345f1388ece#diff-d6ab7dfee8e01c5e0c6d61abd3c7d41c. I get a whole bunch of errors.

Add a comment?

Comment (fregante) [reply source/features/index.tsx:96]: That's not a real page, come on

**Comment by fregante** [comment]
Can you also fix the capitalization here? https://github.com/sindresorhus/onetime/issues/12

File: source/features/comments-time-machine-links.tsx (hunk 3 above)
Comment (yakov116) [reply source/features/comments-time-machine-links.tsx:125]: Comments wont help much. I think the classes are self explanatory

Comment (fregante) [reply source/features/comments-time-machine-links.tsx:125]: Good. I'll revert

**Comment by yakov116** [comment]
@fregante you can merge. I am going on vacation for a week starting tomorrow. and will have very limited access

**Comment by fregante** [comment]
Enjoy!

**Comment by yakov116** [comment]
Thanks!

**Pull request merged**